clients/cobol-lsp-vscode-extension/src/services/copybook/CopybookMessageHandler.ts

```diff
@@ -60,7 +63,7 @@ function searchCopybook(
       dialectType,
     );
     const allowedExtensions = resolveAllowedExtensions(folderKind, documentUri);
-    result = searchCopybookInWorkspace(
+    result = searchCopybookInExtensionFolder(
```

Comment:
Are we backward compatible?

Reply:
The copybooks that are stored under {workspace} /.cz4 is going to be redownloaded after resolving fails if provided settings are valid as before.

Do you see any other possible issues ?

Reply:
The problem is with copybooks that are part of the workspace not the downloaded ones.

Reply:
If you set it with `cobol-lsp.cpy-manager.paths-local setting`. It will search for copybooks firstly from the provided path. 

Do you mean not to ask to user to provide settings for copybooks local path ?

Reply:
It does not work.

Reply:
It is working. only if you provide the absolute path, which I think it is a good idea
For sake of backwards compatibility ;
This is resolved at : a5652c20285e22779fc890ce4cb4275730e7e025 

But the backwards logic wasn't a good idea. Why are we asking local path if we are already looking into workspace folder already ? User should be able to point a path if we ask to provide.

Reply:
Both relative (w.r.t. to workspace) and absolute path should work.

Reply:
should be working with : a5652c20285e22779fc890ce4cb4275730e7e025